css/styles.css

```diff
@@ -858,13 +980,15 @@ fieldset {
   border-radius: 1rem;
   margin-bottom: 1.5rem;
   border: none;
+  z-index: 1000 !important;
 }
 
 .text-area {
   height: 16rem;
 }
 
 .submit {
+  position: relative;
   background-color: #fff;
   color: #0016df;
   font-size: 1.7rem;
```

Comment:
kindly When you create the error message, you should stick to the design guidelines (e.g., font, colors, margins) of [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). and your Background color is not match to the figma template.

Your Design
![Screenshot from 2022-05-20 22-46-51](https://user-images.githubusercontent.com/77842237/169601373-33df1b88-2ecd-4917-87cb-629b01ce1ec2.png)
Figma
![Screenshot from 2022-05-20 22-53-19](https://user-images.githubusercontent.com/77842237/169601684-4d0cd683-c18d-48fa-a91e-9523ef556f75.png)

Reply:
## @maury18  can you please change the background-color of your form-section specially at line number `923` in your CSS file?according to the project requirement even the color should match with the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)